### ISSUE TITLE
Error: asyncio: Cancelling an overlapped future failed

### ISSUE THREAD

**Issue by PriskaH** [simonepri/fitbit2garmin#3]
First of all, many thanks for providing such a useful tool @simonepri !

Unfortunately, I get the following error when running an export with Python version 3.11.6 and 3.12.0:
```
D:\Programs\Python\Python311>fitbit2garmin dump-all -s 2023-10-01 -e 2023-10-03
INFO:root:[1/1] Fetching weight data for 2023-10-01:2023-10-03.
INFO:root:[1/1] No weight data for 2023-10-01:2023-10-03 found.
INFO:root:[1/1] Fetching activity data for 2023-10-01:2023-10-03.
INFO:root:[1/1] Fetched activity data for 2023-10-01:2023-10-03.
INFO:root:Fetching activity log list.
INFO:root:Activity log list fetched.
INFO:root:[1/2] Fetching activity 58926852091.
INFO:root:[1/2] Activity 58926852091 fetched.
INFO:root:[2/2] Activity 58979814656 would have an empty tcx, skipping.
ERROR:asyncio:Cancelling an overlapped future failed
future: <_OverlappedFuture pending cb=[_ProactorReadPipeTransport._loop_reading()]>
Traceback (most recent call last):
  File "D:\Programs\Python\Python311\Lib\asyncio\windows_events.py", line 71, in _cancel_overlapped
    self._ov.cancel()
OSError: [WinError 6] Das Handle ist ungültig
```

I am using a Windows system. I have already tried it with and without the end date parameter, with the same result.

Do you have any idea?

Many thanks,
Priska

**Comment by simonepri** [comment]
Hi @PriskaH, sorry for the late reply. Were you able to run this at the end or is this still relevant?

**Comment by PriskaH** [comment]
Hi @simonepri ,
The issue is still relevant. The result is always the same as described in the latest issue "Activity & Weight Data are missing file extension #4". When I run the command line as admin, I see the error message mentioned in my initial issue description.
Many thanks, 
Priska

**Comment by simonepri** [comment]
> The result is always the same as described in the latest issue "Activity & Weight Data are missing file extension

Can you provide a screen of the produced folder? (e.g. by running `dir /s` in the folder where you run the tool).

**Comment by PriskaH** [comment]
Here it is (it is also the same result when running the tool for a shorter timeframe):

> 18.11.2023  18:28                 0 activity.2015-06-01
18.11.2023  18:31                 0 activity.2015-07-01
18.11.2023  18:35                 0 activity.2015-08-01
18.11.2023  18:39                 0 activity.2015-09-01
18.11.2023  18:42                 0 activity.2015-10-01
18.11.2023  18:46                 0 activity.2015-11-01
18.11.2023  18:50                 0 activity.2015-12-01
18.11.2023  18:53                 0 activity.2016-01-01
18.11.2023  18:57                 0 activity.2016-02-01
18.11.2023  19:01                 0 activity.2016-03-01
18.11.2023  19:04                 0 activity.2016-04-01
18.11.2023  19:08                 0 activity.2016-05-01
18.11.2023  19:12                 0 activity.2016-06-01
18.11.2023  19:16                 0 activity.2016-07-01
18.11.2023  19:19                 0 activity.2016-08-01
18.11.2023  19:23                 0 activity.2016-09-01
18.11.2023  19:27                 0 activity.2016-10-01
18.11.2023  19:30                 0 activity.2016-11-01
18.11.2023  19:34                 0 activity.2016-12-01
18.11.2023  19:38                 0 activity.2017-01-01
18.11.2023  19:41                 0 activity.2017-02-01
18.11.2023  19:45                 0 activity.2017-03-01
18.11.2023  19:49                 0 activity.2017-04-01
18.11.2023  19:52                 0 activity.2017-05-01
18.11.2023  19:56                 0 activity.2017-06-01
18.11.2023  20:00                 0 activity.2017-07-01
18.11.2023  20:03                 0 activity.2017-08-01
18.11.2023  20:07                 0 activity.2017-09-01
18.11.2023  20:11                 0 activity.2017-10-01
18.11.2023  20:14                 0 activity.2017-11-01
18.11.2023  20:18                 0 activity.2017-12-01
18.11.2023  20:22                 0 activity.2018-01-01
18.11.2023  20:25                 0 activity.2018-02-01
18.11.2023  20:29                 0 activity.2018-03-01
18.11.2023  20:33                 0 activity.2018-04-01
18.11.2023  20:36                 0 activity.2018-05-01
18.11.2023  20:40                 0 activity.2018-06-01
18.11.2023  20:44                 0 activity.2018-07-01
18.11.2023  20:47                 0 activity.2018-08-01
18.11.2023  20:51                 0 activity.2018-09-01
18.11.2023  20:55                 0 activity.2018-10-01
18.11.2023  20:58                 0 activity.2018-11-01
18.11.2023  21:02                 0 activity.2018-12-01
18.11.2023  21:06                 0 activity.2019-01-01
18.11.2023  21:09                 0 activity.2019-02-01
18.11.2023  21:13                 0 activity.2019-03-01
18.11.2023  21:17                 0 activity.2019-04-01
18.11.2023  21:20                 0 activity.2019-05-01
18.11.2023  21:24                 0 activity.2019-06-01
18.11.2023  21:28                 0 activity.2019-07-01
18.11.2023  21:31                 0 activity.2019-08-01
18.11.2023  21:35                 0 activity.2019-09-01
18.11.2023  21:39                 0 activity.2019-10-01
18.11.2023  21:43                 0 activity.2019-11-01
18.11.2023  21:46                 0 activity.2019-12-01
18.11.2023  21:50                 0 activity.2020-01-01
18.11.2023  21:54                 0 activity.2020-02-01
18.11.2023  21:57                 0 activity.2020-03-01
18.11.2023  22:01                 0 activity.2020-04-01
18.11.2023  22:05                 0 activity.2020-05-01
18.11.2023  22:08                 0 activity.2020-06-01
18.11.2023  22:12                 0 activity.2020-07-01
18.11.2023  22:16                 0 activity.2020-08-01
18.11.2023  22:19                 0 activity.2020-09-01
18.11.2023  22:23                 0 activity.2020-10-01
18.11.2023  22:27                 0 activity.2020-11-01
18.11.2023  22:30                 0 activity.2020-12-01
18.11.2023  22:34                 0 activity.2021-01-01
18.11.2023  22:38                 0 activity.2021-02-01
18.11.2023  22:41                 0 activity.2021-03-01
18.11.2023  22:45                 0 activity.2021-04-01
18.11.2023  22:49                 0 activity.2021-05-01
18.11.2023  22:52                 0 activity.2021-06-01
18.11.2023  22:56                 0 activity.2021-07-01
18.11.2023  23:00                 0 activity.2021-08-01
18.11.2023  23:03                 0 activity.2021-09-01
18.11.2023  23:07                 0 activity.2021-10-01
18.11.2023  23:11                 0 activity.2021-11-01
18.11.2023  23:14                 0 activity.2021-12-01
18.11.2023  23:18                 0 activity.2022-01-01
18.11.2023  23:22                 0 activity.2022-02-01
18.11.2023  23:25                 0 activity.2022-03-01
18.11.2023  23:29                 0 activity.2022-04-01
18.11.2023  23:33                 0 activity.2022-05-01
18.11.2023  23:37                 0 activity.2022-06-01
18.11.2023  23:40                 0 activity.2022-07-01
18.11.2023  23:44                 0 activity.2022-08-01
18.11.2023  23:48                 0 activity.2022-09-01
18.11.2023  23:51                 0 activity.2022-10-01
18.11.2023  23:55                 0 activity.2022-11-01
18.11.2023  23:59                 0 activity.2022-12-01
19.11.2023  00:02                 0 activity.2023-01-01
19.11.2023  00:06                 0 activity.2023-02-01
19.11.2023  00:10                 0 activity.2023-03-01
19.11.2023  00:13                 0 activity.2023-04-01
19.11.2023  00:17                 0 activity.2023-05-01
19.11.2023  00:21                 0 activity.2023-06-01
19.11.2023  00:24                 0 activity.2023-07-01
19.11.2023  00:28                 0 activity.2023-08-01
19.11.2023  00:32                 0 activity.2023-09-01
19.11.2023  00:35                 0 activity.2023-10-01
19.11.2023  17:41                 0 activity.2023-11-01
19.11.2023  05:58         1.191.926 exercise.10181590333.tcx
19.11.2023  06:02           909.818 exercise.10422243616.tcx
19.11.2023  06:22           649.868 exercise.12707607068.tcx
19.11.2023  06:29         1.381.489 exercise.13455457019.tcx
19.11.2023  06:30         1.118.578 exercise.13618812718.tcx
19.11.2023  06:31         1.235.163 exercise.13685223227.tcx
19.11.2023  07:14         1.013.200 exercise.21511775066.tcx
19.11.2023  07:14         1.131.444 exercise.21556203368.tcx
19.11.2023  07:14         1.479.273 exercise.21694081828.tcx
19.11.2023  07:16         1.202.530 exercise.22031818669.tcx
19.11.2023  07:16         4.348.678 exercise.22117602621.tcx
19.11.2023  07:17         3.198.461 exercise.22139776916.tcx
19.11.2023  07:17         5.499.741 exercise.22140360674.tcx
19.11.2023  07:18         1.307.272 exercise.22348173365.tcx
19.11.2023  07:23         1.099.743 exercise.22840862746.tcx
19.11.2023  07:23         2.146.559 exercise.22964891389.tcx
19.11.2023  07:26           167.447 exercise.23492238965.tcx
19.11.2023  07:28         1.079.592 exercise.23658785877.tcx
19.11.2023  07:32         3.192.655 exercise.24126892911.tcx
19.11.2023  07:32         1.555.010 exercise.24153040636.tcx
19.11.2023  07:34         1.018.725 exercise.24275633293.tcx
19.11.2023  07:37         3.321.259 exercise.24701741724.tcx
19.11.2023  07:39         1.377.159 exercise.25262248780.tcx
19.11.2023  07:40         3.976.871 exercise.25440417313.tcx
19.11.2023  07:41         1.169.270 exercise.25620985998.tcx
19.11.2023  07:45         5.256.317 exercise.26420777303.tcx
19.11.2023  07:46         1.242.700 exercise.26584081362.tcx
19.11.2023  02:45         1.824.541 exercise.2702772944.tcx
19.11.2023  02:45         1.251.599 exercise.2703726606.tcx
19.11.2023  07:49         1.921.353 exercise.27279699654.tcx
19.11.2023  07:50           775.189 exercise.27357212218.tcx
19.11.2023  07:50         1.368.625 exercise.27430188547.tcx
19.11.2023  02:46         1.874.913 exercise.2745179531.tcx
19.11.2023  07:51           608.867 exercise.27459490170.tcx
19.11.2023  02:46         2.739.649 exercise.2746123650.tcx
19.11.2023  02:46         2.073.760 exercise.2747118413.tcx
19.11.2023  08:03         1.224.879 exercise.29873689187.tcx
19.11.2023  08:03         1.123.637 exercise.29951107341.tcx
19.11.2023  08:04         1.227.884 exercise.30154314530.tcx
19.11.2023  08:04         1.301.969 exercise.30381211514.tcx
19.11.2023  08:04         1.300.515 exercise.30492951529.tcx
19.11.2023  08:06         1.217.329 exercise.31144653869.tcx
19.11.2023  08:07        10.391.809 exercise.31199871680.tcx
19.11.2023  08:07         2.572.099 exercise.31402756696.tcx
19.11.2023  08:08         1.234.405 exercise.31477221999.tcx
19.11.2023  08:09         1.108.406 exercise.31662495485.tcx
19.11.2023  08:09         1.204.929 exercise.31747934084.tcx
19.11.2023  08:11         3.066.528 exercise.31869372834.tcx
19.11.2023  08:11         4.074.548 exercise.31904699437.tcx
19.11.2023  08:11         9.001.121 exercise.31907275816.tcx
19.11.2023  08:15         1.212.835 exercise.32778760815.tcx
19.11.2023  08:16         1.143.583 exercise.32902891368.tcx
19.11.2023  08:16         1.123.613 exercise.32954350420.tcx
19.11.2023  08:17         1.132.138 exercise.33075063668.tcx
19.11.2023  08:18         1.102.953 exercise.33120050083.tcx
19.11.2023  08:18         1.636.794 exercise.33157304953.tcx
19.11.2023  08:18         1.118.452 exercise.33181429848.tcx
19.11.2023  08:20         1.102.559 exercise.33503199340.tcx
19.11.2023  08:20         1.151.282 exercise.33505551346.tcx
19.11.2023  08:21         3.543.359 exercise.33863921455.tcx
19.11.2023  08:22         4.060.017 exercise.33952655838.tcx
19.11.2023  08:22         6.130.691 exercise.34053378659.tcx
19.11.2023  08:29         3.268.512 exercise.35551059904.tcx
19.11.2023  08:29         4.118.687 exercise.35668511159.tcx
19.11.2023  08:29         1.620.284 exercise.35701993125.tcx
19.11.2023  08:31         2.220.365 exercise.35850653632.tcx
19.11.2023  08:34           396.074 exercise.36340802995.tcx
19.11.2023  08:34         3.191.324 exercise.36379991111.tcx
19.11.2023  08:38         3.297.730 exercise.37721238429.tcx
19.11.2023  08:41         4.850.645 exercise.38071566845.tcx
19.11.2023  03:29           650.270 exercise.3871678080.tcx
19.11.2023  03:30           649.554 exercise.3892541613.tcx
19.11.2023  03:30           389.461 exercise.3906084603.tcx
19.11.2023  03:30           195.288 exercise.3916430561.tcx
19.11.2023  03:31           554.305 exercise.3937713149.tcx
19.11.2023  08:49         5.700.904 exercise.40106247612.tcx
19.11.2023  08:50        13.411.063 exercise.40132566371.tcx
19.11.2023  08:50        10.452.805 exercise.40184754961.tcx
19.11.2023  08:50         4.856.819 exercise.40231092714.tcx
19.11.2023  08:54         1.942.640 exercise.40976828000.tcx
19.11.2023  08:53         1.606.939 exercise.40978491662.tcx
19.11.2023  08:54         3.345.891 exercise.41014866405.tcx
19.11.2023  08:54         4.403.436 exercise.41016390670.tcx
19.11.2023  08:59         3.595.964 exercise.42309503863.tcx
19.11.2023  08:59         9.369.610 exercise.42370034453.tcx
19.11.2023  08:59         6.886.964 exercise.42374396875.tcx
19.11.2023  09:00        19.422.678 exercise.42433582494.tcx
19.11.2023  09:00        10.799.494 exercise.42481823926.tcx
19.11.2023  09:02         1.257.163 exercise.42733165098.tcx
19.11.2023  03:44           997.559 exercise.4293403747.tcx
19.11.2023  09:05        16.212.266 exercise.43223366341.tcx
19.11.2023  09:04        12.509.580 exercise.43235389291.tcx
19.11.2023  09:05         4.540.588 exercise.43433891170.tcx
19.11.2023  09:06        13.151.299 exercise.43538639464.tcx
19.11.2023  03:47           920.613 exercise.4369346428.tcx
19.11.2023  09:07         3.869.681 exercise.43982262445.tcx
19.11.2023  09:07         5.935.150 exercise.44199016499.tcx
19.11.2023  09:08         1.657.054 exercise.44432440196.tcx
19.11.2023  09:14         4.641.862 exercise.45072923881.tcx
19.11.2023  09:15         6.848.274 exercise.45176153220.tcx
19.11.2023  03:53         1.028.933 exercise.4537261924.tcx
19.11.2023  03:53         1.081.582 exercise.4546785166.tcx
19.11.2023  03:54         1.053.140 exercise.4561164039.tcx
19.11.2023  03:54           939.853 exercise.4573645005.tcx
19.11.2023  03:55           339.960 exercise.4632137684.tcx
19.11.2023  09:22         3.251.051 exercise.46709835238.tcx
19.11.2023  09:22         2.304.398 exercise.46741002170.tcx
19.11.2023  09:23         3.013.142 exercise.47346928518.tcx
19.11.2023  09:25         1.355.333 exercise.47859443114.tcx
19.11.2023  09:26         3.021.708 exercise.47978264738.tcx
19.11.2023  09:28         1.334.616 exercise.48339059269.tcx
19.11.2023  09:30         2.367.079 exercise.48722784669.tcx
19.11.2023  09:30         5.069.569 exercise.48758658554.tcx
19.11.2023  09:31        15.343.672 exercise.48762637860.tcx
19.11.2023  09:31        12.153.483 exercise.48820139799.tcx
19.11.2023  09:31        13.332.830 exercise.48825191792.tcx
19.11.2023  04:04           813.658 exercise.4907671646.tcx
19.11.2023  09:35        10.899.612 exercise.49482081634.tcx
19.11.2023  09:35        20.032.538 exercise.49483747473.tcx
19.11.2023  04:06         1.130.767 exercise.4983024038.tcx
19.11.2023  09:39        10.592.039 exercise.50318322432.tcx
19.11.2023  09:41         1.271.223 exercise.51269891191.tcx
19.11.2023  04:09         1.038.562 exercise.5135934528.tcx
19.11.2023  09:42         1.187.396 exercise.51467911824.tcx
19.11.2023  09:47         1.267.852 exercise.52853055370.tcx
19.11.2023  09:47         9.334.149 exercise.52928707206.tcx
19.11.2023  09:49           736.763 exercise.52956430732.tcx
19.11.2023  09:48         8.950.107 exercise.52958713184.tcx
19.11.2023  09:48         9.545.606 exercise.52961003987.tcx
19.11.2023  09:53         6.574.571 exercise.53559759049.tcx
19.11.2023  04:15         1.103.835 exercise.5382297584.tcx
19.11.2023  04:18         2.063.982 exercise.5483925391.tcx
19.11.2023  09:59        12.906.571 exercise.55458827629.tcx
19.11.2023  10:01         1.051.843 exercise.55914959286.tcx
19.11.2023  10:02         4.469.759 exercise.56244182868.tcx
19.11.2023  10:03        10.184.745 exercise.56311590603.tcx
19.11.2023  10:03        10.155.669 exercise.56337490191.tcx
19.11.2023  10:05           837.511 exercise.56794762415.tcx
19.11.2023  10:05         1.379.007 exercise.56865607623.tcx
19.11.2023  10:06         1.865.176 exercise.56866346284.tcx
19.11.2023  10:06         2.075.239 exercise.56926628395.tcx
19.11.2023  10:07         3.184.388 exercise.56996579011.tcx
19.11.2023  10:08         5.275.454 exercise.56998921627.tcx
19.11.2023  04:24         1.070.791 exercise.5701868743.tcx
19.11.2023  10:09         2.378.830 exercise.57055479378.tcx
19.11.2023  10:08        19.285.721 exercise.57059635627.tcx
19.11.2023  10:09         7.880.223 exercise.57066578072.tcx
19.11.2023  10:10         9.962.830 exercise.57160748931.tcx
19.11.2023  10:11         3.693.016 exercise.57382289195.tcx
19.11.2023  10:15         9.774.577 exercise.58135070314.tcx
19.11.2023  10:15        17.767.958 exercise.58225814919.tcx
19.11.2023  10:17        11.800.509 exercise.58926852091.tcx
19.11.2023  10:19         4.651.908 exercise.59356963201.tcx
19.11.2023  04:29         1.062.800 exercise.6054976122.tcx
19.11.2023  04:35         1.089.165 exercise.6411040814.tcx
19.11.2023  04:38           985.216 exercise.6609917609.tcx
19.11.2023  04:41         1.087.111 exercise.6735309926.tcx
19.11.2023  04:42         1.261.271 exercise.6785760017.tcx
19.11.2023  04:44         1.085.639 exercise.6837338610.tcx
19.11.2023  04:45         1.060.317 exercise.6876821118.tcx
19.11.2023  04:46         1.079.508 exercise.6922019027.tcx
19.11.2023  04:50         1.113.591 exercise.7073427033.tcx
19.11.2023  04:51         1.241.718 exercise.7145135555.tcx
19.11.2023  04:51           841.183 exercise.7153864670.tcx
19.11.2023  04:52         1.067.334 exercise.7228186407.tcx
19.11.2023  04:53         1.178.986 exercise.7250315843.tcx
19.11.2023  04:54         1.095.801 exercise.7276719012.tcx
19.11.2023  04:54           714.478 exercise.7296178551.tcx
19.11.2023  04:56         1.016.305 exercise.7383327327.tcx
19.11.2023  04:57         1.058.209 exercise.7401721938.tcx
19.11.2023  04:59           785.034 exercise.7501086975.tcx
19.11.2023  05:01         1.490.656 exercise.7571312450.tcx
19.11.2023  05:02           985.417 exercise.7634841893.tcx
19.11.2023  05:03           824.090 exercise.7648370952.tcx
19.11.2023  05:03         1.161.776 exercise.7657825481.tcx
19.11.2023  05:04         1.101.572 exercise.7708680127.tcx
19.11.2023  05:05         1.520.219 exercise.7750215666.tcx
19.11.2023  05:05         1.324.678 exercise.7782757862.tcx
19.11.2023  05:07           928.540 exercise.7843201897.tcx
19.11.2023  05:08           942.317 exercise.7878952367.tcx
19.11.2023  05:09           747.981 exercise.7915058893.tcx
19.11.2023  05:10         1.137.794 exercise.7924057211.tcx
19.11.2023  05:11         1.116.195 exercise.7971659236.tcx
19.11.2023  05:12         1.059.470 exercise.8016511214.tcx
19.11.2023  05:13         1.183.752 exercise.8025960387.tcx
19.11.2023  05:13         1.143.075 exercise.8069366757.tcx
19.11.2023  05:14           376.948 exercise.8167675444.tcx
19.11.2023  05:15         1.401.065 exercise.8178246205.tcx
19.11.2023  05:15           769.504 exercise.8197964371.tcx
19.11.2023  05:16         1.115.513 exercise.8237003828.tcx
19.11.2023  05:17         1.091.248 exercise.8290767167.tcx
19.11.2023  05:18         1.151.657 exercise.8309078710.tcx
19.11.2023  05:19         1.431.464 exercise.8347654204.tcx
19.11.2023  05:21         1.353.415 exercise.8424905593.tcx
19.11.2023  05:22         1.329.571 exercise.8454179921.tcx
19.11.2023  05:24         1.049.083 exercise.8561462567.tcx
19.11.2023  05:25           770.501 exercise.8600105176.tcx
19.11.2023  05:27           791.694 exercise.8651158941.tcx
19.11.2023  05:28           833.235 exercise.8674655267.tcx
19.11.2023  05:31         1.020.242 exercise.8778938322.tcx
19.11.2023  05:32           807.812 exercise.8866282514.tcx
19.11.2023  05:35           776.246 exercise.8962546679.tcx
19.11.2023  05:36           899.651 exercise.9063445062.tcx
19.11.2023  05:39         1.093.239 exercise.9161185591.tcx
19.11.2023  05:40           981.676 exercise.9196179617.tcx
19.11.2023  05:42         1.067.381 exercise.9264962760.tcx
19.11.2023  05:44         1.118.196 exercise.9316972119.tcx
19.11.2023  05:51           329.409 exercise.9570929462.tcx
19.11.2023  05:51           965.325 exercise.9675708809.tcx
18.11.2023  17:43                 0 weight.2015-06-01
18.11.2023  17:43                 0 weight.2015-07-01
18.11.2023  17:44                 0 weight.2015-08-01
18.11.2023  17:44                 0 weight.2015-09-01
18.11.2023  17:44                 0 weight.2015-10-01
18.11.2023  17:45                 0 weight.2015-11-01
18.11.2023  17:45                 0 weight.2015-12-01
18.11.2023  17:46                 0 weight.2016-01-01
18.11.2023  17:46                 0 weight.2016-02-01
18.11.2023  17:47                 0 weight.2016-03-01
18.11.2023  17:47                 0 weight.2016-04-01
18.11.2023  17:47                 0 weight.2016-05-01
18.11.2023  17:48                 0 weight.2016-07-01
18.11.2023  17:49                 0 weight.2016-08-01
18.11.2023  17:49                 0 weight.2016-10-01
18.11.2023  17:50                 0 weight.2016-12-01
18.11.2023  17:53                 0 weight.2017-07-01
18.11.2023  17:54                 0 weight.2017-10-01
18.11.2023  18:01                 0 weight.2019-03-01
18.11.2023  18:02                 0 weight.2019-04-01
18.11.2023  18:02                 0 weight.2019-05-01
18.11.2023  18:02                 0 weight.2019-06-01
18.11.2023  18:04                 0 weight.2019-10-01
18.11.2023  18:04                 0 weight.2019-11-01
18.11.2023  18:05                 0 weight.2019-12-01
18.11.2023  18:11                 0 weight.2021-02-01
18.11.2023  18:11                 0 weight.2021-03-01
18.11.2023  18:11                 0 weight.2021-04-01
18.11.2023  18:12                 0 weight.2021-05-01
18.11.2023  18:13                 0 weight.2021-08-01
18.11.2023  18:22                 0 weight.2023-06-01

**Comment by simonepri** [comment]
Thank you for the infos. I just fixed #4 but I could not reproduce the 'Cancelling an overlapped future failed' so for now I'm going to close this issue, but feel free to re-open if you have still issues.

**Comment by PriskaH** [comment]
> Thank you for the infos. I just fixed #4 but I could not reproduce the 'Cancelling an overlapped future failed' so for now I'm going to close this issue, but feel free to re-open if you have still issues.

It is now working for me as well. Many thanks!
